contracts/strategies/pangolin/strategy-png-sherpa-png.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "../strategy-png-farm-base.sol";
```

Comment:
A test and update may be necessary for the strategy-png-farm-base. May not work as intended for png-assetX-png strategies.